### DISCUSSION TITLE
"get_orders" does not return complete information

### DISCUSSION THREAD

**Discussion by alvarosalinero** [saleweaver/python-amazon-sp-api#567] [General]
Hello,

I write because using the get_orders function I can't get the buyer's information.

I have given all the access permissions but I am not getting any results.

What I can do?

Thank you.

All the best,

Alvaro

**Comment by keyurk-emipro** [comment]
Hi @alvarosalinero ,

Calling get_orders method with restricted resources parameter will return Buyer's Information if you have given Tokens API permission.
Orders(...).get_orders( ... ,RestrictedResources=['buyerInfo', 'shippingAddress'])

I hope this will help you.
